presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -0,0 +1,30 @@
+==========
+SHOW STATS
+==========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW STATS FOR table [ WHERE condition ]
```

Comment:
This was my proposal for changing the syntax of the command, but it turns out we don't want this, so we should document the command as it works today:
```
SHOW STATS FOR table
SHOW STATS FOR ( SELECT * FROM table [ WHERE condition ] )
```
Since column projections don't work correctly, we shouldn't document them (we might fix them for now by simply disallowing anything other than `*`).